setup.py

```diff
@@ -19,10 +19,10 @@
 Programming Language :: Python :: 2
 Programming Language :: Python :: 2.7
 Programming Language :: Python :: 3
-Programming Language :: Python :: 3.5
 Programming Language :: Python :: 3.6
 Programming Language :: Python :: 3.7
 Programming Language :: Python :: 3.8
```

Comment:
Not sure whether we should only list versions we are explicitly testing against (so this would match .travis.yml list).